Задачи написаны, но во многих есть элементы подчеркнутые красным. Не …

Lidia128 opened the pull request:
…помогу определить причину. Прошу помочь.

Lidia128 committed:
Задачи написаны, но во многих есть элементы подчеркнутые красным. Не помогу определить причину. Прошу помочь.

Lidia128 committed:
Задачи написаны, но во многих есть элементы подчеркнутые красным. Не помогу определить причину. Прошу помочь.

iStemmer commented on src/Auto/categore/Categore.java:
```diff
@@ -0,0 +1,7 @@
+
+import java.sql.Driver;
+
+public class Categore {}
```
Здесь у нас пустой класс, какова цель его создания?

iStemmer replied:
неактуально

iStemmer commented on src/Product/ProductList.java:
```diff
@@ -0,0 +1,40 @@
+        }
+    }
+    public void checkProduct(String name){
+        for (Product product : this.products){
```
у тебя правильно переопределен hashCode & equals - нет нужды в переборе всего списка, тут можно использовать contains.
Или я не понял для чего нужна эта функция. 
Допустим, для отметки что он уже куплен, но я уже вижу правильный метод по недобавлению

iStemmer commented on src/teacher/Main.java:
```diff
@@ -0,0 +1,46 @@
+    public static LinkedList<Integer>list2= getNumberEnd();
+    public static LinkedList<Integer> stringSet = new ArrayList<>();
+
+    public static void main(String[] args){
```
здесь мне кажется, проще будет переделать
должен быть один сет - который хранит примеры (можно создать класс примера, который превращается d 2*2 или 5*6) 
Дальше - дело техники, в этом примере, как с продуктами переопределяется hashCode&equals или пишется проверка чтобы одинаковые примеры исключать.

Lidia128 committed:
1. Класс Categorу в машинах необходим для связи всех котоегорий с классом Driver.
2. Не могу понять как я могу свять название рецепта и продуктов, которые в него входят
3. Подкорректировала задачу с примерами, но почему-то в рандоме я могу отразить только одно число, если вношу второе, то подчеркивает как ошибку
4.Паспорт поправила.

Lidia128 committed:
2. создала еще один roductList, положила в него другие продукты и создала множество рецептов и положила туда созданные рецепты
3. скорректировала задачу с примерами

Lidia128 committed:
Домашнее заданиее по ассоциативным массивом и картам

iStemmer commented on src/list/ListMap.java:
```diff
@@ -0,0 +1,36 @@
+    public Set<Map.Entry<String,List<Integer>>> getOriginalMapEntries (){
+        return listMap.entrySet();
+    }
+    public Map<String,Integer> getTransformedCollection (){
```
используй форматирование. Ctrl+Alt+L
Такое неудобно читать и неудобно писать
